deployment/common/changeset/mcms_test_helpers.go

```diff
@@ -106,10 +110,15 @@ func ExecuteProposal(t *testing.T, env deployment.Environment, executor *mcms.Ex
 				}
 				tx, err := timelock.ExecuteBatch(
 					env.Chains[sel].DeployerKey, calls, pred, salt)
-				require.NoError(t, err)
+				if err != nil {
+					return err
```

Comment:
return err in attempt to decode the obtuse failure

Reply:
yeah sadly this won't work. The deep reason is that the timelock contract throws away the return data (which has the error) here https://github.com/smartcontractkit/ccip-owner-contracts/blob/9d81692b324ce7ea2ef8a75e683889edbc7e2dd0/src/RBACTimelock.sol#L340